src/operator-manual/telemetry/tracing/01-quickstart.md

```diff
@@ -248,7 +103,7 @@ Then we have to install the Custom Resource Definitions (CRDs) defined by
 Kubewarden:
 
 ```console
-helm install --create-namespace -n kubewarden kubewarden-crds kubewarden/kubewarden-crds
+helm install kubewarden-crds kubewarden/kubewarden-crds
```

Comment:
While this works, Helm saves the release info into the defined workspace (in this case, probably `default`). We may want to keep installing the crds in the same `kubewarden` ns, and open an issue for all notions of it everywhere, for consistency.

Reply:
You are right, fixing.